file_browser/src/file_browser/local/partials/1-elements/_buttons.scss

```diff
@@ -0,0 +1,59 @@
+[class*=regular-button] {
```

Comment:
This selector is never applied. I think what you meant to do is `[class*=button]` to apply this style as the baseline for any class name including the sequence `button`